phenol-core/src/main/java/org/monarchinitiative/phenol/ontology/similarity/PrecomputingPairwiseResnikSimilarity.java

```diff
@@ -64,8 +63,7 @@
    * @param termToIc {@link Map} from{@link TermId} to its information content.
    * @param numThreads Number of threads to use for precomputation.
    */
-  public PrecomputingPairwiseResnikSimilarity(
-    Ontology ontology, Map<TermId, Double> termToIc, int numThreads) {
+  public PrecomputingPairwiseResnikSimilarity(Ontology ontology, Map<TermId, Double> termToIc, int numThreads) {
```

Comment:
Thank-you!

Reply:
Get used to this kind of format for records, though!